scripts/pi-hole/js/queries.js

```diff
@@ -193,13 +193,19 @@ $(document).ready(function() {
             case "4":
                 blocked = true;
                 color = "red";
-                fieldtext = "Pi-holed <br class='hidden-lg'>(wildcard)";
-                buttontext = "";
+                fieldtext = "Blocked <br class='hidden-lg'>(regex/wildcard)";
+                buttontext = "<button style=\"color:green; white-space: nowrap;\"><i class=\"fa fa-pencil-square-o\"></i> Whitelist</button>" ;
                 break;
             case "5":
                 blocked = true;
                 color = "red";
-                fieldtext = "Pi-holed <br class='hidden-lg'>(blacklist)";
+                fieldtext = "Blocked <br class='hidden-lg'>(blacklist)";
+                buttontext = "<button style=\"color:green; white-space: nowrap;\"><i class=\"fa fa-pencil-square-o\"></i> Whitelist</button>" ;
+                break;
+            case "6":
+                blocked = true;
+                color = "red";
+                fieldtext = "Blocked <br class='hidden-lg'>(external)";
                 buttontext = "<button style=\"color:green; white-space: nowrap;\"><i class=\"fa fa-pencil-square-o\"></i> Whitelist</button>" ;
```

Comment:
Externally blocked domains can't be whitelisted if they are blocked in the upstream.

Reply:
@Mcat12 I decided to remove the button for now to avoid presenting a feature that isn't there (as of yet).